.github/workflows/tests.yml

```diff
@@ -19,6 +19,7 @@ permissions:
 
 jobs:
   windows:
+    if: github.event_name == 'push'
```

Comment:
Just leaving a comment for visibility -- we decided to limit this to push events to reduce the exact duplication (on PRs) of the test configuration with and without coverage enabled. We can revisit this once we're ready to start testing more configurations on github actions.